small/small.c

```diff
@@ -223,7 +320,7 @@ smfree_delayed(struct small_alloc *alloc, void *ptr, size_t size)
 {
 	if (alloc->free_mode == SMALL_DELAYED_FREE && ptr) {
 		struct mempool *pool = mempool_find(alloc, size);
-		if (pool == NULL) {
+		if (unlikely(pool == NULL)) {
```

Comment:
That's an optimization and is not relevant to current commit. Please move it to separate commit.